pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 99
```

Comment:
Shall we go for 119, [as in Scrapy](https://github.com/scrapy/scrapy/pull/4237)?